app/views/users/show.html.erb

```diff
@@ -7,10 +7,14 @@
         <%= @user.name %>
       </h1>
     </section>
+    <section class="stats">
+      <%= render "shared/stats" %>
+    </section>
   </aside>
   <div class="col-md-8">
+    <%= render "follow_form" if logged_in? %>
     <% if @user.microposts.any? %>
-      <h3>Microposts (<%= @user.microposts.count %>)</h3>
+      <h3><%= t ".microposts" %>(<%= @user.microposts.count %>)</h3>
```

Comment:
nó không lấy các microposts từ CSDL.nó thực hiện tính toán trong CSDL.yêu cầu CSDL đếm các microposts với user_id
gọi chiều dài trên mảng bằng hàm count()
cái nay trở nên ko hiệu quả nếu microposts lớn